lib/models/rocket.py

```diff
@@ -22,23 +24,23 @@ class Rocket(BaseModel, frozen=True):
     radius: float = 0.0632
     mass: float = 16.235
     motor_position: float = -1.255
-    power_off_drag: "List[Tuple[float, float]]" = [ 
-        (0.01,0.333865758),
-        (0.02,0.394981721),
-        (0.03,0.407756063)
+    power_off_drag: "List[Tuple[float, float]]" = [
+        (0.01, 0.333865758),
+        (0.02, 0.394981721),
+        (0.03, 0.407756063),
     ]
-    power_on_drag: "List[Tuple[float, float]]" = [ 
-        (0.01,0.333865758),
-        (0.02,0.394981721),
-        (0.03,0.407756063)
+    power_on_drag: "List[Tuple[float, float]]" = [
+        (0.01, 0.333865758),
+        (0.02, 0.394981721),
+        (0.03, 0.407756063),
     ]
     _rocket_option: RocketOptions = PrivateAttr()
 
-    #Optional parameters
-    #TBD: a list of possible tailToNose values must be provided in the api docs
+    # Optional parameters
+    # TBD: a list of possible tailToNose values must be provided in the api docs
```

Comment:
As far as I understand, there are only two options: tail_to_nose or nose_to_tail.
It should be easy to insert both examples in the docs and then delete this TODO.

But I understand if you want to do this one later.

Reply:
That is the goal, but instead of just adding it as an observation in the docs I want to make this a [field](https://fastapi.tiangolo.com/tutorial/body-fields/) validation so it can have a default value and hopefully an auto-generated standardized disclaimer in the API docs.

I think I need to add this to the comment tho.

In the future, the idea is to remove any guard method from the controllers and transform them into field validations to make better use of fasAPI features.